### PR TITLE
Better Looking Darkmode + Update Nav button text

### PR DESCRIPTION
**Pull request by death7654**

### CODE REVIEW

**Review by ninelore — CHANGES_REQUESTED** [review]
Blocking for some Checks i need to do

**Comment by death7654** [comment]
> Blocking for some Checks i need to do

Why its only css

**Comment by ninelore** [comment]
SEO

**Comment by ninelore** [comment]
SEO

**Comment by ethanaobrien** [comment]
> Blocking for some Checks i need to do

Are these "checks" related to the css that was actually changed, and/or related to the css already in the project?

**Comment by MrChromebox** [comment]
Again, what does "better" mean? it's meaningless word. Tell us exactly what you are changing and why

File: src/.vuepress/styles/index.scss [hunk 1]
```diff
@@ -84,3 +87,12 @@ html.dark {
 div.theme-default-content:not(.custom), footer.page-meta:not(.custom), nav.page-nav:not(.custom) {
   max-width: 1800px;
 }
+
```
Comment (MrChromebox) [inline src/.vuepress/styles/index.scss:90]: how does this change relate to "better" dark mode?

Comment (death7654) [reply src/.vuepress/styles/index.scss:90]: line 90?

Comment (ninelore) [reply src/.vuepress/styles/index.scss:90]: formatting and i dont see an issue including that when modifying the file

**Comment by ninelore** [comment]
The markdown changes are still in this branch/PR

**Comment by death7654** [comment]
I will be redoing this, in a new branch closing

File: src/.vuepress/styles/index.scss (hunk 1 above)
Comment (ethanaobrien) [reply src/.vuepress/styles/index.scss:90]: > GH doesn't allow you to select multiple lines to comment on

I recently learned it does actually, try holding shift while clicking on any additional lines

File: src/.vuepress/styles/index.scss [hunk 2]
```diff
@@ -84,3 +87,12 @@ html.dark {
...
+.deviceSearch
+{
+  all: unset;
+  background-color: var(--c-bg);
+  border-radius: 10px;
+  border: var(--border) solid 1px;
+  padding-left: 10px;
+}
```
Comment (MrChromebox) [inline src/.vuepress/styles/index.scss:98]: not sure it's selecting multiple lines here

Comment (MrChromebox) [reply src/.vuepress/styles/index.scss:98]: looks like it did actually, nice that they fixed that

**Pull request closed without merging**